weave/cache.py

```diff
@@ -95,7 +96,15 @@ def clear_cache() -> None:
                 shutil.rmtree(item_path)
                 logging.info(f"Deleted {item}.")
         except:
-            logging.info(f"Error deleting {item}.")
+            # oftentimes we will except to here if the item is not a timestamp
```

Comment:
It would be preferable to specifically catch the error which you are trying to handle, as opposed to a general exception. Moreover, why do we have these non-timestamp directories anyway? Is this just legacy? If not, then is this a safe operation?

Reply:
maybe your, right, i might just create another script to wipe out all the legacy stuff rather than bake it into this one